runtime/altair/src/lib.rs

```diff
@@ -1020,7 +1045,7 @@ parameter_types! {
 }
 
 impl pallet_permissions::Config for Runtime {
-	type AdminOrigin = EnsureRootOr<HalfOfCouncil>;
+	type AdminOrigin = EnsureRootOr<EitherOf<HalfOfCouncil, PoolAdmin>>;
```

Comment:
OpenGov introduces a `PoolAdmin` track just for pool related stuff. Of course, this includes pool creation. IMO it also makes sense to include the permissions and keystore `AdminOrigin`. I would be interested in your thoughts @mustermeiszer @hieronx. For Altair, it does not matter too much probably but in the end Altair and Centrifuge should have the same configuration apart from the track parameters (deposits, durations, voting thresholds).

The difference between the `Root` and `PoolAdmin` tracks is that the latter has much lower thresholds in terms of deposits, support and it is faster:

Root
<img width="642" alt="Screenshot 2024-05-14 at 15 16 49" src="https://github.com/centrifuge/centrifuge-chain/assets/26296386/35279eb4-f849-409c-89fb-41101d521e60">

Pool Admin
<img width="646" alt="Screenshot 2024-05-14 at 15 16 18" src="https://github.com/centrifuge/centrifuge-chain/assets/26296386/78cb280b-0035-455f-bdd4-3dca1180e09a">